frontend/react/src/App.js

```
@@ -1,24 +1,39 @@
+/* 
+To Enable OKTA Authentication, you will need to:
+1. Uncomment out the OKTA Enabled section below
+2. Comment out the OKTA Disabled section below
+3. Go to layout > Header.js and ensure the Logout feature is uncommented
+
+To Disable OKTA Authentication, you will need to:
+1. Uncomment out the OKTA Disabled section below
+2. Comment out the OKTA Enabled section below
+3. Got to layout > Header.js and ensure the Logout feature is commented out
+*/
+
 import React from "react";
 import "font-awesome/css/font-awesome.min.css";
 import "./App.scss";
-import Routes from "./reactRouter";
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { Security, SecureRoute, LoginCallback } from '@okta/okta-react';
+import Home from "./Home";
+import config from './auth-config';
 import Header from "./components/layout/Header";
 import Footer from "./components/layout/Footer";
 import InitialDataLoad from "./components/Utils/InitialDataLoad";
 
 function App() {
-  let VisibleHeader =
-    window.location.pathname.split("/")[1] === "reports" ? null : <Header />;
-
-  let VisibleFooter =
-    window.location.pathname.split("/")[1] === "reports" ? null : <Footer />;
   return (
-    <div className="App" data-test="component-app">
-      <InitialDataLoad />
-      {VisibleHeader}
-      <Routes />
-      {VisibleFooter}
-    </div>
+    <InitialDataLoad />
+    <Router>
+      {/* OKTA Enabled */}
+      {/* <Security {...config.oidc}>
```

Default: Show `<Security/>`, hide `<Route component={Home}/>`
Flag present: Hide `<Security/>`, show `<Route component={Home}/>`